bdba/client.py

```diff
@@ -611,3 +611,28 @@ def bdio_export(
                 entries=raw_data.get('@graph'),
             ),
         )
+
+    # SBOM-Export-Methods 
+    def cyclonedx_sbom_export(self, product_id: int | str) -> dict:
```

Comment:
nit nit: let's keep consistent code-style across this repository, therefore please use one argument per line.
```suggestion
    def cyclonedx_sbom_export(
      self,
      product_id: int | str,
    ) -> dict:
```